packages/sdks/snippets/hydrogen/CHANGELOG.md

```diff
@@ -0,0 +1,529 @@
+# skeleton
```

Comment:
can we remove the CHANGELOG.md, it might cause confusion for users visiting this snippet project